Binary files duplicity-0.4.9.orig/.DS_Store and duplicity-0.4.9/.DS_Store differ
diff -urN duplicity-0.4.9.orig/setup.py duplicity-0.4.9/setup.py
--- duplicity-0.4.9.orig/setup.py	2008-01-04 18:11:39.000000000 +0000
+++ duplicity-0.4.9/setup.py	2008-02-17 00:20:04.000000000 +0000
@@ -48,8 +48,7 @@
 							   library_dirs=libdir_list,
 							   libraries=["rsync"])],
 	  scripts = ['rdiffdir', 'duplicity'],
-	  data_files = [('share/man/man1', ['duplicity.1', 'rdiffdir.1']),
-					('share/doc/duplicity-%s' % version_string,
-					 ['COPYING', 'README', 'CHANGELOG'])])
+	  data_files = [('share/man/man1', 
+			 ['duplicity.1', 'rdiffdir.1'])])
 
 
